### ISSUE TITLE
Implementation of Stochastic Encoder

### ISSUE THREAD

**Issue by shreejalt** [LEGO999/BigBiGAN-TensorFlow2.0#3]
Hi, 
How can we implement a stochastic encoder in your code? As far as I know, the Stochastic encoder is just the variational Bayes using a KLD Loss Function. Correct me if I am wrong.

Thanking you
